lib/test/cdo/test_log_collector.rb

```diff
@@ -0,0 +1,41 @@
+require_relative '../test_helper'
+require 'cdo/log_collector'
+
+class LogCollectorTest < Minitest::Test
+  def test_time_a_function
```

Comment:
Can we use `TimeCop` to test that LogCollector correctly measures the amount of time it took the function to execute?

Reply:
Will keep this idea for future improvement. 
I'm looking at examples of Timecop in our code base; 2 common functions are`Timecop.freeze(time)` and `Timecop.travel` and it is not obvious to me how to measure execution time?

Reply:
Makes sense.  I think we could freeze time before invoking a method that is wrapped in the log collector, and inside the wrapped method we could travel forward.

Reply:
Got it!